WISE-Community/WISE
# Revived StudentServiceImplTest

How to test:
1. Run ```mvn test```. 190 tests should run and pass.
1. Open StudentServiceImplTest.java in VSCode. There should be a "Run | Debug" link above the class definition if you have installed Java Debugger plugin. Click on Run. All 6 tests in the class should run and pass.
1. Try running or debugging few of the test methods. They should pass.

Closes #2068
